src/clickable-marker-map/ClickableMarkerMap.js

```diff
@@ -0,0 +1,137 @@
+import { geoAlbers } from "d3-geo";
+import { scaleSqrt } from "d3-scale";
+import PropTypes from "prop-types";
+import React from "react";
+import {
+  ComposableMap,
+  Geographies,
+  Geography,
+  Marker,
+} from "react-simple-maps";
+import styled from "styled-components";
+import { mesh } from "topojson";
+import { THEME, TOTAL_KEY } from "../constants";
+
+const MAX_MARKER_RADIUS = 19;
+
+const LocationMarker = styled.circle`
+  cursor: pointer;
+  fill: ${(props) =>
+    props.selected
+      ? props.theme.mapMarkers.hover.fill
+      : props.theme.mapMarkers.default.fill};
+  fill-opacity: ${(props) => props.theme.mapMarkers.default.fillOpacity};
+  stroke: ${(props) => props.theme.mapMarkers.default.stroke};
+  stroke-width: ${(props) => props.theme.mapMarkers.default.strokeWidth};
+  transition: ${(props) => props.theme.mapMarkers.default.transition};
+
+  &:hover {
+    fill: ${(props) => props.theme.mapMarkers.hover.fill};
+  }
+
+  &:active {
+    fill: ${(props) => props.theme.mapMarkers.pressed.fill};
+  }
+`;
+
+export default function ClickableMarkerMap({
```

Comment:
This is the map that underlies the Prison Population and Parole Population maps.  I don't really care for the names `ClickableMarkerMap` and `ClickableRegionMap`, but they are if nothing else, quite literal.  If we don't like the idea of extracting these components based upon their interaction style, then you can stop reading here and we can revert this commit.

Reply:
it's not a problem for now but I am worried that this division may prove too prescriptive, i.e. if we have a need for a map that combines both interaction styles. My thought on this had been to extend `StateMap` to also support clickable markers, since I don't think there is really any reason they need to be mutually exclusive. (It's possible that wouldn't have worked well in practice? I didn't take it farther than a vague idea.)